### ISSUE TITLE
focus outline not visible on video and gallery link

### ISSUE THREAD

**Issue by zebapy** [middlebury/midd-frontend#136]
May be due to absolute positioned elements

**Comment by zebapy** [comment]
We have the focus style (blue overlay) now so, although it's not an outline, it should suffice.
